examples/bh_threaded.c

```diff
@@ -13,6 +13,14 @@
 #include <linux/gpio.h>
 #include <linux/delay.h>
 #include <linux/interrupt.h>
+#include <linux/version.h>
+
+/* gpio_request_array() and gpio_free_array() 
+ * were removed in kernel v6.10-rc1 .
+ */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 10, 0)
+#define NO_ARRAY
```

Comment:
The macro identifier is misleading. Refine its name.

Reply:
I have replaced `NO_ARRAY` with `GPIO_NO_REQUEST_ARRAY` to make it clearer.